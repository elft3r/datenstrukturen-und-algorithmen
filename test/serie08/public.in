7
EU
DJN
DQOBS
LGAUWQU
BELDXTDE
MFULMDPUWA
GIYXKECLGIBY
PAVRRNXKGSXCAC
TFDLVQFHECUDKXE
SMTYXEBJLFAOTNTPUOXT
PBFUDVNWQYNJWJDYUAGPOAXJP
CTSBXDNMSCNRQSHLPGHNOMJAWHPMCX
SKDTMFSMMPCQGIPWMHNTBCVUQKOFNJMPFFG
SMGHBBPEURBWORAYONXITQXDTEVTLHRBPPXDWBQG
