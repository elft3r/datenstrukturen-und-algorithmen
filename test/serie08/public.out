0 
1 Q
2 LD
3 GXC
5 TLFUX
9 BDNQNJWPX
10 SMGPWNTBQG
